secure/key-manager/src/tests.rs

```diff
@@ -79,6 +99,7 @@ fn setup(config: &NodeConfig) -> Node {
 
 #[derive(Clone)]
 struct TestLibraInterface {
+    queued_transactions: Arc<RefCell<Vec<Transaction>>>,
```

Comment:
why arc? I thought RefCell is enough?

Reply:
We clone it for use within node and within key manager. I thought RefCell would be enough too until the test failed :D.